lib/fika/type_checker.ex

```diff
@@ -197,6 +197,12 @@ defmodule Fika.TypeChecker do
     end
   end
 
+  # Atom value
+  def infer_exp(env, {:atom, _line, atom}) do
+    Logger.debug("Atom value #{atom} found. Type: atom")
```

Comment:
The type will be `#{atom}` instead of `atom` because we don't have an "atom" type, instead, the atom's value itself is the type here. For example, type of "123" is String but type of :ok is just :ok.

Reply:
I see! So it will become like this?
```suggestion
    Logger.debug("Atom value found. Type: #{atom}")
```

Reply:
Yes!